src/main/java/net/minecraftforge/event/terraingen/WorldTypeEvent.java

```diff
@@ -69,9 +82,9 @@ public WorldType getWorldType()
         private final int originalSize;
         private int newSize;
 
-        public BiomeSize(WorldType worldType, int original)
+        public BiomeSize(@Nullable World world, WorldType worldType, int original)
```

Comment:
Public methods like this can't be changed directly, they need to be deprecated and replaced with new methods.